### DISCUSSION TITLE
Laravel Livewire npm run dev

### DISCUSSION THREAD

**Discussion by alan-su-jetfuel** [serversideup/spin#123] [General]
Use spin create new project that run success. Laravel homepage can saw but now need to run `npm run dev` or `npm run build` for developer.

1. Faild for command
```
spin exec npm run build
spin exec npm run dev
```

2. Try to retstart node docker that still not build about vite files

How to suggestion for developer hot reload vite? Thank.

**Comment by jaydrogers** [comment]
We have hot reload and HTTPS support for Vite with Spin Pro https://getspin.pro/
